src/main/java/seedu/address/MainApp.java

```diff
@@ -73,24 +67,39 @@ public void init() throws Exception {
      * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
      */
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
-        logger.info("Using data file : " + storage.getAddressBookFilePath());
+        logger.info("Using data file : " + storage.getVolunteerStorageFilePath());
```

Comment:
are we using 2 files for separate volunteer and event storage?

Reply:
better for testing!